test/quantization/test_quantize_script.py

```diff
@@ -2625,3 +2654,224 @@ def forward(self, x):
         for x, obs in model._modules._c.items():
             graph_params.append((obs.getattr('3_scale_0'), obs.getattr('3_zero_point_0')))
         self.assertEqual(ref_qparams, graph_params)
+class TestQuantizeScript(QuantizationTestCase):
```

Comment:
please include new line before the class, wonder why there is no lint for this one